client/me/purchases/manage-purchase/index.jsx

```diff
@@ -171,14 +171,18 @@ const ManagePurchase = React.createClass( {
 		}
 		if ( isMonthly( purchase.productSlug ) ) {
 			const expiryMoment = this.moment( purchase.expiryMoment );
-			const daysToExpiry = this.moment( expiryMoment.diff( this.moment() ) ).format( 'D' );
+			const daysToExpiry = Number( this.moment( expiryMoment.diff( this.moment() ) ).format( 'D' ) );
 
-			return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s days. ',
+			return this.translate(
+				'%(purchaseName)s will expire and be removed from your site in %(daysToExpiry)d day. ',
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( '%(purchaseName)s will expire and be removed from your site in %(daysToExpiry)s day. ', '%(purchaseName)s will expire and be removed from your site in %(daysToExpiry)s days. ' )` *ES Score: 6.42*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).